x-pack/plugins/security/server/usage_collector/security_usage_collector.ts

```diff
@@ -59,7 +59,7 @@ export function registerSecurityUsageCollector({ usageCollection, config, licens
         type: 'boolean',
       },
       authProviderCount: {
-        type: 'number',
+        type: 'long',
```

Comment:
Realistically, we wouldn't ever have enough auth providers in Kibana to warrant a "long". But we don't enforce an actual limit, so it's probably the most appropriate type to use.